packages/wallet-sdk/src/components/ConnectContent/ConnectContent.tsx

```diff
@@ -8,12 +8,10 @@ import { useCallback, useState } from 'preact/hooks';
 import { createQrUrl } from '../../util';
```

Comment:
before
![Screenshot 2023-10-27 at 4 59 15 PM](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/a9b85cef-fe9d-45a3-b212-60718c12ff49)

after
![image](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/5fcd0881-6666-4521-b445-9b532e5c8978)

Reply:
fyi, we are working with design for new UI, removing the white space and making it responsive to screensize.